.gitlab-ci.yml

```diff
@@ -102,9 +102,35 @@ download-single-step-artifacts:
 package-oci:
   needs: [ download-single-step-artifacts ]
 
-generate-lib-init-pinned-tag-values:
+package-oci-musl:
+  extends: package-oci
+  parallel:
+    matrix:
+      - ARCH: amd64
+        LIBC: musl
+
+create-arch-specific-lib-injection-image-musl:
+  extends: create-arch-specific-lib-injection-image
+  needs: [ package-oci-musl ]
+  parallel:
+    matrix:
+      - ARCH: musl # technically not true but that's how the package is named
+        DOCKER_PLATFORM: linux/amd64
+  script:
+    - export VERSION="$(<sources/version)"
+    - mv packaging/datadog-apm-library-dotnet-${VERSION}-linux-amd64.tar packaging/datadog-apm-library-dotnet-${VERSION}-linux-musl.tar
```

Comment:
I wonder if you need to include `ARCH` in here somewhere? 🤔

Reply:
no, that's handled in `create-arch-specific-lib-injection-image`'s script which this references. By having the `ARCH` as `musl` it just works.